RecoParticleFlow/PFClusterProducer/python/particleFlowRecHitECAL_cfi.py

```diff
@@ -15,10 +16,11 @@
            cms.PSet(
              name = cms.string("PFEBRecHitCreator"),
              src  = cms.InputTag("ecalRecHit","EcalRecHitsEB"),
+             srFlags = cms.InputTag("ecalDigis"),
              qualityTests = cms.VPSet(
                   cms.PSet(
-                  name = cms.string("PFRecHitQTestThreshold"),
-                  threshold = cms.double(0.08)
+                  name = cms.string("PFRecHitQTestECALThreshold"),
+                  thresholds = particle_flow_zero_suppression_ECAL.thresholds
```

Comment:
less efficient version at run time will be as outlined in https://github.com/cms-sw/cmssw/pull/17794#pullrequestreview-28737441

a more efficient one would be manipulating the qualityTests VPset and for that one I can't write an working example without actually checking it; it will be something similar to 
https://github.com/cms-sw/cmssw/blob/master/RecoParticleFlow/PFProducer/python/particleFlowBlock_cfi.py#L139